dev.sh

```diff
@@ -18,8 +18,12 @@ docker run -d \
 	--name reverse-proxy \
 	reverse-proxy 443 localhost
 
+# the real dhparam will be 4096 bits but that takes ages to generate
```

Comment:
can you indicate where the real dhparam is generated?

Reply:
Updated README